File: samples/client/petstore/java/src/main/java/org/openapijsonschematools/client/apis/paths/Userlogin.java
package org.openapijsonschematools.client.apis.paths;

import org.openapijsonschematools.client.apiclient.ApiClient;
import org.openapijsonschematools.client.configurations.ApiConfiguration;
import org.openapijsonschematools.client.configurations.SchemaConfiguration;
import org.openapijsonschematools.client.paths.userlogin.Get;

public class Userlogin extends ApiClient implements
    Get.GetOperation
{
    public Userlogin(ApiConfiguration apiConfiguration, SchemaConfiguration schemaConfiguration) {
        super(apiConfiguration, schemaConfiguration);
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/apis/paths/Userlogin.kt
package org.openapijsonschematools.client.apis.paths;

import org.openapijsonschematools.client.apiclient.ApiClient;
import org.openapijsonschematools.client.configurations.ApiConfiguration;
import org.openapijsonschematools.client.configurations.SchemaConfiguration;
import org.openapijsonschematools.client.paths.userlogin.UserloginGet;

public class Userlogin extends ApiClient implements
    UserloginGet.GetOperation
{
    public Userlogin(ApiConfiguration apiConfiguration, SchemaConfiguration schemaConfiguration) {
        super(apiConfiguration, schemaConfiguration);
    }
}